.nvmrc

```diff
@@ -1 +1 @@
-v14.18.1
+v16
```

Comment:
do we need this file?

Reply:
Yes that's why I updated. If you have nvm setup to pick up the `.nvmrc` file automatically and we remove this you may equally have an error if you default version is not what we expect.